### PR TITLE
Vacms 10263 qa refactor

### PR DESCRIPTION
**Pull request by timcosgrove**
## Description
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10263
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10258

### CODE REVIEW

File: src/data/queries/linkTeaser.ts [hunk 1]
```diff
@@ -30,17 +30,16 @@ export const data: QueryData<DataOpts, ParagraphLinkTeaser[]> = async (
   return entities
 }
 
-export const formatter: QueryFormatter<
-  ParagraphLinkTeaser[],
-  LinkTeaserType[]
-> = (entities: ParagraphLinkTeaser[]) => {
-  return entities.map((entity) => ({
+export const formatter: QueryFormatter<ParagraphLinkTeaser, LinkTeaserType> = (
```
Comment (timcosgrove) [inline src/data/queries/linkTeaser.ts:33]: I reworked this one to only operate on one entity. In most cases, I would expect a data formatter to format a single entity.

File: src/data/queries/questionAnswer.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+export const params: QueryParams<null> = () => {
+  return queries.getParams().addInclude([
+    'field_answer',
+    'field_buttons',
+    'field_related_benefit_hubs',
+    'field_related_information',
+    'field_tags.field_topics',
+    'field_tags.field_audience_beneficiares',
+    'field_tags.field_non_beneficiares',
+    // 'field_administration',
```
Comment (timcosgrove) [inline src/data/queries/questionAnswer.ts:23]: These will be added when we have components and formatters for outputting these fields.

File: src/pages/[[...slug]].tsx [hunk 3]
```diff
@@ -7,8 +7,13 @@ import { getGlobalElements } from '@/lib/context/getGlobalElements'
 import { Wrapper } from '@/templates/globals/wrapper'
```
Comment (timcosgrove) [inline src/pages/[[...slug]].tsx:7]: The file renaming from [...slug] to [[...slug]] allows the page to respond to `/` requests (i.e. root index). We're not doing anything with it, but it makes it possible that the index page will eventually be managed by Drupal.

**Review by ndouglas — APPROVED** [review]

File: src/data/queries/audienceTopics.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import { ParagraphAudienceTopics } from '@/types/dataTypes/drupal/paragraph'
+import { QueryFormatter } from 'next-drupal-query'
+import { AudienceTopicType } from '@/types/index'
+
+const getTagsList = (fieldTags) => {
+  if (!fieldTags) return null
+
+  const {
+    field_topics: fieldTopics = [],
+    field_audience_beneficiares: fieldAudienceBeneficiares,
```
[the remark this answers is not in the document]
Comment (timcosgrove) [reply src/data/queries/audienceTopics.ts:11]: It was actually the source of a couple bugs for us.

**Pull request merged**